decidim-surveys/spec/system/surveys_breadcrumbs_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Surveys Breadcrumb" do
+  include_context "with a component"
+
+  let(:manifest_name) { "surveys" }
+  let(:title) do
```

Comment:
The survey cells is displaying the titles escaped ... does not know about the trick we've done with `generate_component_name` names.